bin/run-browser-tests-ci

```diff
@@ -12,6 +12,7 @@ docker run \
   -v "$(pwd)/browser-test/bin:/usr/src/civiform-browser-tests/bin" \
   --network browser-test_default \
   civiform-browser-test:latest \
-  /usr/src/civiform-browser-tests/bin/wait_for_server_start_and_run_tests.sh
+  /usr/src/civiform-browser-tests/bin/wait_for_server_start_and_run_tests.sh \
+  $@
```

Comment:
allows passing a specific test filename through since we don't want to run the whole suite for azure tests, just the ones that exercise azure integration